lib/manageiq/environment.rb

```diff
@@ -92,8 +85,6 @@ def self.seed_database
     end
 
     def self.setup_test_environment(task_prefix: '', root: APP_ROOT)
-      return if ENV["SKIP_TEST_RESET"]
```

Comment:
I moved this out intentionally, so the guts don't have to know about ENV vars, and also it's more consistent with the other methods that don't check for ENV vars from within the method.